src/data/analyses.js

```diff
@@ -0,0 +1,121 @@
+var MAP = {
```

Comment:
in the worst case (the key not existing because it has not been updated) a non-interpolated string would appear, from `.getAnalysisByType`, that's the case for "Source" analyses

it's difficult to decouple from cartodb, because in the `data/analyses.js` there's stuff regarding backbone forms, for example https://github.com/CartoDB/cartodb/blob/master/lib/assets/core/javascripts/cartodb3/data/analyses.js#L16

what would be your proposal?